pandas/core/groupby/generic.py

```diff
@@ -1141,7 +1141,7 @@ def aggregate(self, func=None, *args, engine=None, engine_kwargs=None, **kwargs)
                     result = gba.agg()
 
                 except ValueError as err:
-                    if "no results" not in str(err):
+                    if "No objects to concatenate" not in str(err):
                         # raised directly by _aggregate_multiple_funcs
```

Comment:
is this comment still accurate?  why can't we get rid of the try/except in this case?

Reply:
Thanks - comment wasn't accurate, I've removed it. Unlike cases where we just skip over columns that fail, this code falls back to a different method that can then succeed. As far as I can tell, this can only be hit by an empty DataFrame (and in particular, I think a DataFrame with no columns).

In general I plan on looking at this pattern (try A except B) throughout groupby and seeing where it can be removed in followups.

Reply:
awesome.  whatever you determine, can you follow up with a useful comment here